osbrain/tests/test_proxy.py

```diff
@@ -119,6 +122,51 @@ def ping(self):
         InitTimeoutProxy('foo', timeout=1.)
 
 
+@pytest.mark.parametrize('timeout', [-1, 1])
+def test_agent_proxy_wait_running(nsproxy, timeout):
+    """
+    Using `wait_for_running` on a proxy after initialization should block until
+    the agent is running or time out.
+    """
+    AgentProcess('agent').start()
+
+    # Get "offline" proxy
+    agent = Proxy('agent')
+    time0 = time.time()
+    Timer(abs(timeout)/2, agent.run).start()
+
+    proxy = Proxy('agent').wait_for_running(timeout=timeout)
+    assert proxy.ping() == 'pong'
+    assert abs(timeout)/2 <= time.time() - time0 <= abs(timeout)
```

Comment:
I would rather always keep spaces around operators (i.e.: around the `/` here and in line 136). But I see the PEP8 and Flakes tests not complaining so...

Anyway it could be better, for consistency with the rest of the code.